includes/admin.form.inc

```diff
@@ -141,7 +141,7 @@ function islandora_repository_admin(array $form, array &$form_state) {
         'islandora_deny_inactive_and_deleted' => array(
           '#type' => 'checkbox',
           '#title' => t('Lock down inactive and deleted objects.'),
-          '#description' => t('Deny access to inactive or deleted objects using a separate permission than for active objects.'),
+          '#description' => t('Require a separate permission to access inactive and deleted objects.'),
```

Comment:
So, question: we can apply an exception here to make sure this all goes into the release, but does a change in a form element's description qualifies as documentation or improvement? Could be nice to add to our workflows so this type of improvements make its way into the release after code freeze (and encourage them?) @Islandora/7-x-1-x-committers thoughts?

Reply:
Descriptions are is 100% in-app documentation.